src/main/java/com/sjsu/backbenchers/vBless/controller/CampaignRestController.java

```diff
@@ -132,7 +134,7 @@
 	public ResponseEntity<Campaign> updateCampaignStatus(@RequestParam Long campaignId){
 		log.debug("updateCampaignStatus.... " + campaignId);
 		Campaign campaign = campaignRepository.findOne(campaignId);
-		campaign.setActive("N");
+		//campaign.setActive("N");
```

Comment:
We need to remove this later may be, is there any reason we commented out campaign.setActive("N"); now?